src/components/Navigation/styles.scss

```diff
@@ -0,0 +1,117 @@
+// Mobile portrait
+.wrapper {
+	width: 100%;
+	height: 100%;
+	background-color: red;
+}
+
+	.button {
+		display: inline-flex;
+		position: fixed;
+		top: 1.15em;
+		right: 1em;
+		align-items: center;
+		font-size: 1.5rem;
+		
+		&:before {
```

Comment:
```suggestion
		&::before {
```
pseudo-element!

Reply:
Ah it worked with one : but looks like that's CSS2 and :: is CSS3.
Changed anyway so it's most up to date.